mappings/net/minecraft/client/render/entity/BipedEntityRenderer.mapping

```diff
@@ -1 +1,9 @@
 CLASS net/minecraft/class_1147 net/minecraft/client/render/entity/BipedEntityRenderer
+	FIELD field_4863 bipedModel Lnet/minecraft/class_866;
```

Comment:
does yarn use the Biped name?

Reply:
the `model` field is already declared. so i prefixed each with the class prefix